### DISCUSSION TITLE
Would it be too much to ask to include a static library for each release?

### DISCUSSION THREAD

**Discussion by edyu** [duckdb/duckdb#8117] [General]
I know the current release (at least for the c/c++ version) of libduckdb has duckdb.h, duckdb.hpp, and libduckdb.so. If the point of duckdb is to be an in-process database, I think it makes sense to have a static library libduckdb.a included in the release.

**Comment by Mause** [comment]
This was already being built, just not uploaded. I've opened https://github.com/duckdb/duckdb/pull/8197

**Reply by edyu** [reply]
Thank you. Found it!
